README.md

```diff
@@ -47,6 +47,9 @@ replication:
   period: 300
   connectionTimeout: 30
   receiveTimeout: 60
+  sites:
```

Comment:
❓ How is this going to work? I was under the impression a worker would be handling 1 or more queues, is it now a Replicator will take a set of destination sites and work those off the queue instead?

Reply:
This ticket is not introducing queues yet. When queues are introduced and we now have workers in play, this property will also be usable to dictate which queues should be monitored by the worker.

I hear you for destination sites. The issue I have is that for workers (in the future), this won't be destination but actually the set of queues to monitor. For tactical sites, this will represent the tactical site from which we will push/pull. So I agree that for the time being, this name might be a bit unclear but I feel we should wait for the design to start changing before we settle on one.

As for the README. I am all for documenting those there and I did comment on exactly that on your PR 😄 . So the question is wether or not we decide to document them.

Reply:
The only thing I'd argue about documenting this property is it is replication specific, while the metrics properties are well documented by spring boot already.

Reply:
I just think we should document all of them. If some properties don't merit documentation then they should probably not be there. What do you think?

Reply:
For non-replication specific properties we can link to their respective documentation, otherwise we should document it.

Reply:
Although, if we expect those to be configurable for our needs, it would be easier to have them documented all in one place I think.